db/seeds.rb

```diff
@@ -6,31 +6,46 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-10.times do
+Proposal.destroy_all
+Job.destroy_all
+User.destroy_all
+
+User.create(
+  username: 'kenji',
+  email: 'teste@gmail.com',
+  skill: 'Portuguese',
+  first_name: 'Alexandre',
+  last_name: 'Tsuchiya',
+  password: "123456",
+  password_confirmation: "123456",
+  admin: true
+)
+
+200.times do
   User.create(
     username: Faker::Internet.username,
     email: Faker::Internet.email,
     skill: Faker::Nation.language,
     first_name: Faker::Name.first_name,
-    last_name: Faker::Name.last_name
+    last_name: Faker::Name.last_name,
+    password: "123456",
+    password_confirmation: "123456"
   )
-  puts "created user"
+  puts "created user #{User.last.username}"
 end
 
-20.times do
+50.times do
   Job.create(
     content: "translate #{Faker::Nation.language} to #{Faker::Nation.language}",
```

Comment:
Insight - para o produto final, pode ser interessante trocar o content por "lingua de origem', 'lingua de destino', 'numero de paginas' e 'campo aberto para comentários' ´

eventualemente, pode ser um string padrão, que puxe esses dados de um formulário. O que for mais fácil.